src/materialized/src/bin/materialized/main.rs

```diff
@@ -588,6 +588,8 @@ fn run(args: Args) -> Result<(), anyhow::Error> {
                     suppress_output: false,
                     process_listen_host: args.process_listen_host,
                     data_dir: args.data_directory.clone(),
+                    command_wrapper: env::var("MZ_PROCESS_ORCHESTRATOR_WRAPPER")
+                        .map_or(Ok(vec![]), |s| shell_words::split(&s))?,
```

Comment:
Can you introduce a new CLI argument `--process-orchestrator-wrapper` that has the env var as a fallback? As a matter of policy we've tried to avoid any env var-only configs.

Reply:
Yup, will try to remember :)